### PR TITLE
Create Orders Bug

### PR DESCRIPTION
**Pull request by kiran-mastek**
Regression - Can no longer create Orders for select orgs

Update AK_OrderingParty_OdsCode as filtered index
Update project version to allow build

Story: [AB#12959](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/12959)
Task: [AB#14971](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/14971)

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/NHSD.BuyingCatalogue.Ordering.OrderingDatabase.sqlproj [hunk 1]
```diff
@@ -101,4 +106,7 @@
       <Value>$(SqlCmdVar__1)</Value>
     </SqlCmdVariable>
   </ItemGroup>
-</Project>
+  <ItemGroup>
+    <None Include="Indexes\OdsCodeIndex.sql" />
+  </ItemGroup>
+</Project>
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/NHSD.BuyingCatalogue.Ordering.OrderingDatabase.sqlproj:112]: Probably VS that has removed it but file should terminate with a new line.

**Review by bjss-chris — CHANGES_REQUESTED** [review]
Changes look good overall – just a couple of very small things.

File: src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Indexes/OdsCodeIndex.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+﻿CREATE UNIQUE NONCLUSTERED INDEX AK_OrderingParty_OdsCode
```
[the remark this answers is not in the document]
Comment (kiran-mastek) [reply src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Indexes/OdsCodeIndex.sql:1]: done

File: src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/NHSD.BuyingCatalogue.Ordering.OrderingDatabase.sqlproj (hunk 1 above)
Comment (kiran-mastek) [reply src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/NHSD.BuyingCatalogue.Ordering.OrderingDatabase.sqlproj]: done

**Review by bjss-chris — APPROVED** [review]

**Pull request merged**